internal/exportimport/importer.go

```diff
@@ -52,6 +55,7 @@ func (s *Server) handleImport() http.Handler {
 			logger.Errorw("unable to read active configs", "error", err)
 		}
 
+		anyErrors := false
```

Comment:
Nit: I'd prefer if we named this "success" and inverted. Then it's "if !success"